### DISCUSSION TITLE
v0.20.4

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#3212] [Versions & Releases]
## What's Changed
* Update XTTS cloning by @erogol in https://github.com/coqui-ai/TTS/pull/3207
* fix max generation length for XTTS by @WeberJulian in https://github.com/coqui-ai/TTS/pull/3208

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.20.3...v0.20.4

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.20.4'>v0.20.4</a>.</em>

**Comment by ccingiz** [comment]
> ## What's Changed
> * Update XTTS cloning by @erogol in https://github.com/coqui-ai/TTS/pull/3207
> * fix max generation length for XTTS by @WeberJulian in https://github.com/coqui-ai/TTS/pull/3208
> 
> 
> **Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.20.3...v0.20.4
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.20.4'>v0.20.4</a>.</em>
